lib/json_schema.dart

```diff
@@ -36,6 +36,8 @@
 //     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //     THE SOFTWARE.
 
+export 'package:json_schema/src/json_schema/custom_vocabularies.dart'
+    show CustomVocabulary, CustomKeyword, ValidationContext;
```

Comment:
Any reason we must give the consumer access to ValidationContext's constructor? Since we already create one for them and they simply return it?